bindings/wasm/examples/src/1_advanced/7_status_list_2021.ts

```
@@ -0,0 +1,157 @@
+    const CREDENTIAL_INDEX = 5;
+    const status = new StatusList2021Entry(statusListCredential.id(), statusListCredential.purpose(), CREDENTIAL_INDEX)
+        .toStatus();
+    const credential = new Credential({
+        id: "https://example.edu/credentials/3732",
+        type: "UniversityDegreeCredential",
+        credentialStatus: status,
+        issuer: issuerDocument.id(),
+        credentialSubject: subject,
+    });
+
+    // Create signed JWT credential.
+    const credentialJwt = await issuerDocument.createCredentialJwt(
+        issuerStorage,
+        issuerFragment,
+        credential,
+        new JwsSignatureOptions(),
+    );
+    console.log(`Credential JWT > ${credentialJwt.toString()}`);
+
+    // Validate the credential using the issuer's DID Document.
+    const validationOptions = new JwtCredentialValidationOptions({ status: StatusCheck.SkipUnsupported });
+    let jwtCredentialValidator = new JwtCredentialValidator(new Ed25519JwsVerifier());
+    jwtCredentialValidator.validate(
+        credentialJwt,
+        issuerDocument,
+        validationOptions,
+        FailFast.FirstError,
+    );
+
+    // ===========================================================================
+    // Revocation of the Verifiable Credential.
+    // ===========================================================================
+
+    // Update the status list credential.
+    // This revokes the credential's unique index.
+    statusListCredential.setCredentialStatus(credential, CREDENTIAL_INDEX, true);
+
+    // Credential verification now fails.
+    try {
+        jwtCredentialValidator.validate(
+            credentialJwt,
+            issuerDocument,
+            validationOptions,
+            FailFast.FirstError,
+        );
+        JwtCredentialValidator.checkStatusWithStatusList2021(credential, statusListCredential, StatusCheck.Strict);
+        console.log("Revocation Failed!");
```

This is a bid ambiguous

---

Copied this from `vc_revocation` example. How can I improve it? Shall I add a comment specifying that since the credential has been revoked `checkStatusWithStatusList2021` should throw an exception?